deps/wazuh_testing/wazuh_testing/modules/fim/event_monitor.py

```diff
@@ -476,6 +488,46 @@ def detect_whodata_start(file_monitor, timeout=T_60):
                        error_message=ERR_MSG_WHODATA_ENGINE_EVENT)
 
 
+def get_messages(callback, timeout=T_30):
+    """Look for as many synchronization events as possible.
+    This function will look for the synchronization messages until a Timeout is raised or 'max_events' is reached.
+    Args:
+        timeout (int): Timeout that will be used to get the dbsync_no_data message.

```

Comment:
Please, add the correct Args.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4220/commits/342572271a42610bdc01315915bc2da74fdb1a83